make_ios.sh

```diff
@@ -38,6 +38,8 @@ cd ios
 
 echo "=> Removing unnecessary files and folders"
 rm -rf {PROJECT_NAME}/sources/data
+rm -f {PROJECT_NAME}KIFUITests/Sources/Specs/Application/ApplicationSpec.swift
```

Comment:
@markgravity, you should give more insight about this 🙏 Why do we have to remove these files and better to have a story in `iOS-template` repo link to this.

Reply:
@phongvhd93 updated it here
[c57b737](https://github.com/nimblehq/kmm-templates/pull/23/commits/c57b737b87048836ec99a50ace8d2b1f99ab67eb)